scripts/travis/android/entrypoint/install.sh

```diff
@@ -5,7 +5,9 @@ host=$1
 abi=$2
 
 scons -Q clean
-scons -Q --compiler=clang \
+
+# remove soversion from soname (for gradle android plugin)
+LDFLAGS="-Wl,-soname,libroc.so" scons -Q --compiler=clang \
```

Comment:
Let's just add an option to scons to disable soversion feature. I'll push it to master soon.

Reply:
Added `--disable-soversion` and pushed to master.

Reply:
The soversion problem is fixed now, but the soname itself is still needed. Changing 
[https://github.com/roc-project/roc/blob/master/SConstruct#L1085-L1089](https://github.com/roc-project/roc/blob/master/SConstruct#L1085-L1089) from
```
if not GetOption('disable_soversion'):
    lib_env['SHLIBSUFFIX'] = '%s.%s' % (lib_env['SHLIBSUFFIX'], abi_version)
    lib_env.Append(LINKFLAGS=[
        '-Wl,-soname,libroc%s' % lib_env['SHLIBSUFFIX'],
    ])
```
to
```
if not GetOption('disable_soversion'):
    lib_env['SHLIBSUFFIX'] = '%s.%s' % (lib_env['SHLIBSUFFIX'], abi_version)
lib_env.Append(LINKFLAGS=[
    '-Wl,-soname,libroc%s' % lib_env['SHLIBSUFFIX'],
])
```
will fix the problem

Reply:
Oops, pushed another fix. BTW, what error do you get without soname? Just curious.